arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,242 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
 
-#### Developer wallet / account
+#### Developer wallet/account
 
-When deploying on and interacting with a testnet chain, it’s important to use a fresh wallet that does not contain any real assets. You’ll often be including private keys as CLI arguments to execute transactions programmatically, **_so avoid using personal accounts for development_**.
+You will often use crypto private keys as CLI arguments when sending transactions on Arbitrum networks, so we recommend creating an account specifically for development that doesn't hold any real assets.
 
-If you’re using [MetaMask](https://metamask.io/), simply click the dropdown at the top middle of the plugin and then click “Add Account” to create a fresh account. It can be helpful to label the account as a dev wallet or “Stylus” for this purpose. You’ll need this newly created account’s private key (as well as some Sepolia ETH) for deploying a smart contract. [Follow the instructions on MetaMask’s website](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to obtain your key.
+##### Creating a new account
 
-![Stylus Wallet](./assets/stylus-wallet.png)
+If you’re using [MetaMask](https://metamask.io/), click the dropdown at the top middle of the plugin and then click “Add Account” to create a fresh account. Labeling the account as a dev wallet or “Stylus” for this purpose can be helpful. You’ll need this newly created account’s private key and some Sepolia ETH to deploy a smart contract. [Follow the instructions on MetaMask’s website](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to obtain your key.
 
-:::caution
+##### Storing your private key
 
-**_Never share your “secret recovery phrase” with anyone_**. Never enter it anywhere. A _private key_ is valid for an individual account only, but a _secret recovery phrase_ can be used to gain access to ALL accounts in your wallet.
+You might also want to add your developer's private key to your environment variables for easier and safer access.
+To do this, store your hex-encoded private key in a text file and add the path to the file as an ENV variable named `PRIVATE_KEY_PATH` to your shell` file .bashrc`, `.zshrc` or other. You can also add this variable to a `.env` file at the root of your project.
 
-:::
+#### Arbitrum Sepolia ETH
 
-### Testnet ETH
+Stylus settles to [Arbitrum Sepolia](./reference/testnet-information.md) testnet, so you will need at least one Sepolia ETH to deploy contracts and send transactions. 
 
-The Stylus testnet settles directly to the [Arbitrum Sepolia](/build-decentralized-apps/03-public-chains.md#arbitrum-sepolia) testnet. Follow these steps to acquire testnet ETH on the Stylus testnet:
+Start by visiting a faucet to obtain Sepolia testnet ETH. Here are a few options:
```

Comment:
There's a quicker path here, which is to directly obtain Arbitrum Sepolia ETH. You have some faucets here, but I'm not sure which ones are active (Quicknode and Alchemy are active for sure): https://arbitrum.faucet.dev/ArbSepolia